keanu-project/src/main/java/io/improbable/keanu/vertices/dbl/nonprobabilistic/ConstantDoubleVertex.java

```diff
@@ -44,7 +65,16 @@ public DoubleTensor sample(KeanuRandom random) {
     }
 
     @Override
+    public Map<String, Vertex> getParentsMap() {
+        return new HashMap<>();
```

Comment:
This behaviour is the same as that in the method you're overriding (in `Vertex`).

Reply:
Yeah - the parent version will become abstract once this is implemented everywhere so I don't want to rely on the base class implementation at the mo.